src/context/__tests__/auth.test.tsx

```
@@ -0,0 +1,753 @@
+    vi.clearAllMocks();
+    authStateChangedCallback = null;
+    
+    // Setup mock for onAuthStateChanged to capture the callback
+    mockOnAuthStateChanged.mockImplementation((callback) => {
+      authStateChangedCallback = callback;
+      // Return unsubscribe function
+      return vi.fn();
+    });
+    
+    // Clear window auth context
+    delete (window as any).authContext;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)